src/common/filter/Expressions.cpp

```diff
@@ -619,6 +621,19 @@ const char* FunctionCallExpression::decode(const char *pos, const char *end) {
     return pos;
 }
 
+std::string UUIDExpression::toString() const {
+    return folly::stringPrintf("uuid(%s)", field_->c_str());
+}
+
+OptVariantType UUIDExpression::eval() const {
+     auto client = context_->storageClient();
+     auto space = context_->space();
+     return client->getUUID(space, *field_);
```

Comment:
The interface in StorageClient is async, maybe you need to get future here.

Reply:
Sure！ `StorageClient`  will be update when #958 merged :)

Reply:
Noted.